core/blockchain.go

```diff
@@ -2009,6 +2023,11 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 			log.Debug("Abort during block processing")
 			break
 		}
+		//For fast node, if the H-11 block has not been verified, stop processing blocks.
```

Comment:
Can we try hide the code here? If you look into the `it.next()`, we will learn that most validation is done there.